pkg/jsonSchemaValidator/extensions/customKeyRegoRule.go

```diff
@@ -0,0 +1,123 @@
+// This file defines a custom key to implement the logic for the rule:
```

Comment:
@TzlilSwimmer123 i think this file should be named "customKeyRegoDefinition" like the others whose name is the same name as the key